### PR TITLE
README for quilt

### PR DESCRIPTION
**Pull request by pgarrison**
# Purpose
Currently our dataset is published with a README.pdf, but Quilt does not render this. I've migrated the README.pdf to a README.md markdown file that Quilt can render.

This documentation is about the dataset, not directly about this code, so technically it doesn't need to be in this repository. I'm putting it here for the following reasons:
1. By hosting `dataset_structure.svg` here, we don't have to have it in the S3 bucket where it could create confusion as an extraneous file
2. Allows us to review changes to docs via pull requests
3. Github will render markdown to preview it before pushing to S3

# Implementation
Since images cannot be embedded directly in markdown files, the image `dataset_structure.svg` is its own file.

To avoid putting the `dataset_structure.svg` file on quilt, the link to it is an absolute link to github.com. As a write this, the link uses the branch name `quilt-readme` because that's where the file exists, but I'll update it to `bioRxiv-v1` before merging this pull request.

### CODE REVIEW

File: docs/quilt/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+CollagenIV segmentation probability | collagenIV_segmentation_probability.ome.zarr 
+CollagenIV segmentaion mesh | collagenIV_segmentation_mesh 
+
+For improved data accessibility we provide further information in the `manifests` section below.
+
+## manifests
+
+The `manifests` directory contains 6 manifests for streamlined data accessibility. Each manifest file and its contents are explained below. 
+
+* imaging_and_segmentation_data.csv: contains both the links to download and links to visualize without download, all the imaging data, segmentation data, and mesh data. This csv also provides columns that contain all accompanying metadata from this study.
```
Comment (vianamp) [inline docs/quilt/README.md]: Can we standardize the capitalization of this CSV names?

Comment (vianamp) [reply docs/quilt/README.md]: And can we put the file names between ` `?

File: docs/quilt/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* Image_analysis_extracted_features_column_description.csv: A complete description of each column label present in the Image_analysis_extracted_features.csv is provided here. 
+
+* Migration_timing_through_mesh_extracted_features.csv: contains class annotations for the centroids of individual nuclei, indicating whether each centroid is inside or outside the CollagenIV_segmentation_mesh. 
+
+* Migration_timing_through_mesh_extracted_features_column_description.csv: A complete description of each column label present in the Migration_timing_through_mesh_extracted_features.csv is provided here. 
+ 
+
+## supplemental_files
+
+The `supplemental_files` directory contains model checkpoints (cytodl_checkpoints) and CytoDL configuration files (cytodl_configuration_files) used for all cells mask and collagenIV mask generation.
```
Comment (vianamp) [inline docs/quilt/README.md]: Isn't there a model for ZO1 segmentation too?

**Review by vianamp — APPROVED** [review]
Looks good.

File: docs/quilt/README.md (hunk 1 above)
Comment (pgarrison) [reply docs/quilt/README.md]: I think @smishra3 was using the capitalization of the CSVs we uploaded?

File: docs/quilt/README.md (hunk 2 above)
Comment (pgarrison) [reply docs/quilt/README.md]: No, we did not include ZO1 segmentation code or data in this release.

File: docs/quilt/README.md (hunk 1 above)
Comment (pgarrison) [reply docs/quilt/README.md]: Added backticks in d68f17d

**Pull request merged**